torch/csrc/distributed/c10d/c10d_frontend.cpp

```diff
@@ -0,0 +1,635 @@
+#include <torch/csrc/distributed/c10d/c10d_frontend.h>
```

Comment:
since there is already a `c10d` in the path, shall we rename this file to just `frontend`?

Reply:
Sure. done.

@wanchaol Wanchao, you will probably want to change file name as well in your diffs.